http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20200915_add_chart_to_app_and_cluster_tables.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="kyuksel" id="add_chart_to_app_table">
+        <addColumn tableName="APP">
+            <column name="chart" type="VARCHAR(254)" defaultValue="dummyChartName-x.y.z">
```

Comment:
shall we run a `insert` cmd in current database, and remove the `defaultValue` here? just so there's no way going forward, a record will be populated with the dummy value

Reply:
That sounds good to me, could do something like:
1. add nullable collumn
2. INSERT statement to set the current values
3. make column not null